changelog/unreleased/issue-3090

```diff
@@ -0,0 +1,6 @@
+Bugfix: --use-fs-snapshot failed on windows/386
+
+restic failed to create VSS snapshots on windows/386 because of invalid API calls.
```

Comment:
"because of invalid API calls" is an implementation detail. I'd include the actual error message that users would have seen, so they can search for it. Also, I think @rawtaz wants Restic with a capital letter at the start of a sentence and you forgot to mention that the issue is fixed now :)

Reply:
You read my mind :D

Reply:
>I'd include the actual error message that users would have seen, so they can search for it.

It's not a good idea: GetSnapshotProperties failed with the same error message for pretty much [everything](https://github.com/restic/restic/pull/2274#issuecomment-683151351).

Reply:
Fixed in https://github.com/restic/restic/pull/3090/commits/88208c3db27da33ab9141f10abf384b7a9045221